h/groups/util.py

```diff
@@ -33,3 +33,7 @@ def pubid(self):
     @property
     def is_public(self):
         return True
+
+    @property
+    def creator_id(self):
+        return None
```

Comment:
Adding stuff like this here is fine for now -- we can merge this PR without addressing this comment -- but I really don't think it's going to be hard to move the `__public__` group into the database now we have an explicit `hypothesis init` command. So perhaps you could do that as a followup so we don't have to keep implementing functionality here as well?